stock_picking_report_cmr/views/report_cmr.xml

```diff
@@ -0,0 +1,1170 @@
+                                                <td style="width:10%;" />
+                                            </tr>
+
+                                            <tr style="height:11px !important;">
+                                                <td
+                                                    style="width:30%;border-left:0px ! important;"
+                                                > Total to be paid: </td>
+                                                <td style="width:12%;" />
+                                                <td style="width:10%;" />
+                                                <td style="width:12%;" />
+                                                <td style="width:12%;" />
+                                                <td style="width:12%;" />
+                                                <td style="width:10%;" />
+                                            </tr>
+
+                                        </table>
+                                    </td>
+                                </tr>
+                                <tr style="height: 20mm">
+                                    <td class="w-50" style="border-right:none;">
+                                        <table class="table-inner">
+                                            <tr>
+                                                <td style="font-size: 1.5em">
+                                                    <strong>14</strong>
+                                                </td>
+                                                <td class="pl-1">
+                                                    Istruzioni per il pagamento
+                                                    <br />
+                                                    Prescriptions d'affranchissement
+                                                    <br />
+                                                    Instructions as tu payment for
+                                                    carriage
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td class="border" />
+                                                <td>
+                                                    Porto / Franco / Carriage
+                                                    paid
+                                                </td>
+                                            </tr>
+                                            <tr>
+                                                <td class="border" />
+                                                <td>
+                                                    Porto / Non franco / Carriage
+                                                    forward
+                                                </td>
```

Comment:
```suggestion

```
Better remove these check boxes (not mandatory) and leave space for a simple text field. To be filled in a future improvement of this report.